### ISSUE TITLE
User Guide overhaul

### ISSUE THREAD

**Issue by bzbug** [neomutt/neomutt#3480]
The User Guide right now is closer to a reference than to a guide for beginners. It talks a lot about secondary details and additional features for configuration before the beginner can even set up his mail accounts and try all those features discussed in the guide.

## Expected Behaviour

1. I, a beginner who's only used Web clients from Tech Giants for mail, go to User Guide and the first thing I find there is a `How To Set Up Your Accounts and Start Receiving and Sending Mail` before I read anything else that requires me to already _have_ a mail account set up. Including descriptions of the UI that is only accessible _after_ you set up an account and receive some mail in your client.

2. I expect a `Basic Defaults` config file explained in detail for that purpose and I expect the file to have something that would allow the most standard setup nowdays - mail from multiple accounts that goes into the same folders (for example, mail from `bob@example.com` and `bobby@example.com` goes into `INBOX` folder, but gets tagged somehow with the corresponding mail account it was received from).

3. I expect the most crucial terms like `mailbox` explained explicitly. Right now I'm still not sure a mailbox is just a folder, because there's some option that's called `folder` in neomutt, but the guide calls those `mailboxes`. And also because traditionally, when using Web clients for mail, I've only ever seen the word `mailbox` used to describe _a mail account_. `bob@example.com` is one mailbox which has many folders in it, `bobby@example.com` is another mailbox with its own folders. And how do tags differ from flags? What is `spool file`? The man page doesn't make it any clearer, but it gets mentioned in the User Guide before it gets explained.

## Actual Behaviour

I now know a lot about UI that I can't even see in my actuall neomutt on my PC, because I haven't set up any accounts yet, I know loads about many tiny settings like `index_format` and path shortening for folders in the sidebar and such, but I have no clue how to set up my mail accounts. All that theory that I've read in hopes to _find already_ the explanation of how to set up my accounts, is completely useless because I wasn't even able to _try_ anything out in neomutt while reading. And it's totally unnecessary to start with all those tiny additional features when oftentimes the user wants to be able to use the primary function of the programm as soon as possible and set everything else up later, in his own pace.

The guide is, indeed, not a guide but a reference, it talks about features with the mindset of a neomutt developer, not an actual new user. Hello, beginner, I'll show you full description of UI before you are even able to see that UI with your own eyes. Hello, beginner, this is some hook called `folder-hook`, it's just one of the features of neomutt, not something that you should know about first and foremost if you want to set up your accounts already.

The explanations, the whole Guide should be ordered not in the way that makes sense to a developer of neomutt, but in the way that makes sense to a user of neomutt. The features should be introduced based on most common use-cases, not on the structure of the program, and the most primary function of the program (sending and receiving mail) should be set up and explained before anything else.

---

It would also be nice if someone pointed to me where I can find the explanation of how to set up multiple accounts that would be managed from the same folders (without having to switch between accounts in order to access their mail, the folders should be for all accounts at once, like in all modern Web clients for mail that allow for mail retrieval from other mail accounts via IMAP/POP3).

**Comment by Aym3nJ** [comment]
Yeah the User Guide is outdated and points to  none existing  [sample.neomuttrc-starter](https://github.com/neomutt/neomutt/blob/main/contrib/samples/sample.neomuttrc-starter) file!

**Comment by flatcap** [comment]
@bzbug Make a lot of good points.
The documentation is a mess.

Fixing it would be a huge task.  Someone would have to dedicate a lot of time to it.
And _that's_ the problem -- nobody's prepared to spend that time.
This project has very few developers and we're busy trying to fix bugs.

Theoretically, the docs could be improved by anyone who speaks English and has a familiarity with NeoMutt.
I would happily make time to mentor them.

In practice, hundreds of people write a blog post about how to do **X** in Mutt / NeoMutt, but don't contribute that back to the project.

If you're reading this and you want NeoMutt to improve, **you** need to help.
Anything you can contribute helps to relieve the pressure on the developers and build the NeoMutt community.

---

You can also find us on IRC: `#neomutt` on `irc.libera.chat` ([web client](https://kiwiirc.com/nextclient/irc.libera.chat/#neomutt))

**Comment by bzbug** [comment]
@flatcap, It is a good reminder to those who know how to use NeoMutt already, sometimes even a simple PR that clarifies just one tiny point in the docs is better than no PR at all.

But I wanted to point out that the people who are most invested in good documentation for NeoMutt now (apart from the usual team of contributors) are the people who _don't_ know how to use NeoMutt. I, for instance, am not _able_ to write good documentation right now because in order to explain something, one has to understand it himself.

Just a quick reminder that many people who recommend newcommers writing their own, improved documentation forget that most newcommers can't do that since they can't understand the program without an already existing good documentation ¯\\\_(⊙_ʖ⊙)\_/¯

At this point in time I've postponed the dive into NeoMutt's internal logic and such that would have allowed me understand it, since without good documentation it requires far more time than I have right now. So, I still have no idea how to implement the common web mail logic I've mentioned, and I'm still not using NeoMutt, even though I want it working this whole time at least in the most basic configuration of the common web mail logic.

I understand the struggle and the limited time you guys can invest into this in the current situation, but, just in case there was an illusion that newcommers will do the docs properly themselves, they most likely won't. Because they are newcommers. However, if the docs are improved, it's way more easy for NeoMutt to get more fresh blood, and the new people will probably add to the team of developers with time and ease your burden.
